### PR TITLE
aubo_robot: 0.1.0-1 in 'indigo/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by liuxinwust**
Increasing version of package(s) in repository `aubo_robot` to `0.1.0-1`:
- upstream repository: https://github.com/auboliuxin/aubo_robot.git
- release repository: https://github.com/auboliuxin/aubo_robot-release.git
- distro file: `indigo/distribution.yaml`
- bloom version: `0.5.22`
- previous version for package: `null`
## aubo_control

```
* add CHANGELOG.rst
* update version 0.0.1
* update README
* Aubo robotics ROS version 1.0
* Contributors: robot
```
## aubo_description

```
* add CHANGELOG.rst
* update version 0.0.1
* update README
* Aubo robotics ROS version 1.0
* Contributors: robot
```
## aubo_driver

```
* add CHANGELOG.rst
* update version 0.0.1
* update README
* Aubo robotics ROS version 1.0
* Contributors: robot
```
## aubo_gazebo

```
* add CHANGELOG.rst
* update version 0.0.1
* update README
* Aubo robotics ROS version 1.0
* Contributors: robot
```
## aubo_i5_moveit_config

```
* add CHANGELOG.rst
* Aubo robotics ROS version 1.0
* Contributors: robot
```
## aubo_kinematics

```
* add CHANGELOG.rst
* update version 0.0.1
* update README
* Aubo robotics ROS version 1.0
* Contributors: robot
```
## aubo_msgs

```
* add CHANGELOG.rst
* update version 0.0.1
* update README
* Aubo robotics ROS version 1.0
* Contributors: robot
```
## aubo_robot

```
* add CHANGELOG.rst
* update version 0.0.1
* update README
* Aubo robotics ROS version 1.0
* Contributors: robot
```
## aubo_trajectory

```
* add CHANGELOG.rst
* update version 0.0.1
* update README
* Aubo robotics ROS version 1.0
* Contributors: robot
```

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -466,6 +466,31 @@ repositories:
...
+      - aubo_robot
+      - aubo_trajectory
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/auboliuxin/aubo_robot-release.git
+      version: 0.1.0-1
+    source:
+      type: git
+      url: https://github.com/auboliuxin/aubo_robot.git
+      version: 0.1.0
```
Comment (tfoote) [inline indigo/distribution.yaml]: The source entry needs to be a branch name, so in your case it should be `master`

**Comment by tfoote** [comment]
Hi @auboliuxin I've closed all your other PRs as this one is the closest to passing the tests. 

This release appears to be complete except for the version I commented on above. You can edit this branch to change that one line and the CI will retry.  If you have more questions please be patient and give us an opportunity to reply.

Also if you have pull requests that are out of date or have been replaced please take a moment to close them so it's clearer for us that we don't need to review the replaced pull requests. 

Note that you can run the CI tests yourself if you checkout this branch of your fork of the rosdistro and run the `nosetests -s` in the root.  

You will need [these](https://github.com/ros/rosdistro/blob/master/.travis.yml#L9-L12) dependencies but  I recommend installing them from apt not pip if possible on your platform.

**Comment by dirk-thomas** [comment]
@tfoote If you look at the repository where this PR comes from it says "unknown repository". So I don't think this can be updated anymore.

**Comment by tfoote** [comment]
Thanks @dirk-thomas I hadn't noticed that. 

For others watching github has conveniently created a patch file that's wgetable to keep this PR alive despite the underlying repo dissapearing. `curl -L https://github.com/ros/rosdistro/pull/12747.patch`

Since this branch is not updateable I'm going to close this. 

@auboliuxin The easiest thing for you to do will likely be to rerun bloom-release with the `--pull-request-only ` option which should automatically recreate the necessary fork and open a new pull request.

**Pull request closed without merging**